.circleci/config.yml

```diff
@@ -71,18 +71,22 @@ jobs:
           path: test-reports
           destination: test-reports
 
-  test-3.7-signac-1.0.0:
+  test-3.8-signac-1.0.0:
     <<: *test-template
     environment:
       SIGNAC_VERSION: "v1.0.0"
-  test-3.7-signac-1.1.0:
+  test-3.8-signac-1.2.0:
     <<: *test-template
     environment:
-      SIGNAC_VERSION: "v1.1.0"
-  test-3.7-signac-1.2.0:
+      SIGNAC_VERSION: "v1.2.0"
+  test-3.8-signac-1.3.0:
     <<: *test-template
     environment:
-      SIGNAC_VERSION: "v1.2.0"
+      SIGNAC_VERSION: "v1.3.0"
```

Comment:
I decided to keep 1.0 and the two most recent minor versions -- we could test 1.1 if needed but I think that's overkill in our build matrix.

Reply:
Fair enough. Alternatively we test all minor versions only for release branches?